app/src/app/[lng]/page.tsx

```diff
@@ -403,7 +401,10 @@ export default function Home({ params: { lng } }: { params: { lng: string } }) {
                 fontWeight="semibold"
                 lineHeight="32"
               >
-                GPC Basic Emission Inventory Calculation - Year {year}
+                <Trans t={t}>
+                  gpc-basic-emissions-inventory-calculations-year
+                </Trans>{" "}
+                {year}
```

Comment:
You can have placeholders for data in translation strings: `{{year}}`, and then pass this data into the `t` function or the `Trans` component. Here this works but there are more complex examples of having variables in the middle of text etc. where it becomes relevant.